utils/storage/cookie.js

```diff
@@ -12,6 +12,7 @@ class CookieLocal {
   constructor(options) {
     this._options = {};
     this.options(options);
+    this.cookieSupported = this.IsCookieSupported();
```

Comment:
The class already has 'cookie' in its name. Let's rename the variable to `isSupportAvailable`.

Reply:
Same with the method name.

Reply:
Addressed